src/Raven.Studio/typescript/widgets/virtualGrid/textColumn.ts

```diff
@@ -0,0 +1,25 @@
+﻿import virtualColumn = require("widgets/virtualGrid/virtualColumn");
+
+/**
+ * Virtual grid column that renders text in its cells.
+ */
+class textColumn implements virtualColumn {
+    readonly cellClass = "text-cell";
+
+    constructor(
+        public dataMemberName: string,
+        public display: string,
+        public width: string) {
+    }
+
+    renderCell(item: Object, isSelected: boolean): string {
+        const cellValue = (item as any)[this.dataMemberName];
+        if (cellValue != null) {
```

Comment:
I thought that at first, too. But on closer inspection, we actually want the null check: consider if the value is false-y (e.g. zero), we still want to display it.